### DISCUSSION TITLE
Importing existing files

### DISCUSSION THREAD

**Discussion by MrPresident2** [immich-app/immich#764] [Q&A]
Hey guys,
I'm a first-time user, and I'm trying to import all of my existing files. is there a way to import or get the app to scan a folder from the server or do I need to re-upload all of my data, have duplication, and all that fun stuff.
Thanks!

**Comment by micogg** [comment]
@MrPresident2 Did you find a solution for this?

**Comment by alextran1502** [comment]
@MrPresident2 @micogg Sorry I missed the thread. You guys can use the CLI tool to do that https://github.com/immich-app/CLI

**Reply by micogg** [reply]
Thanks @alextran1502 

I may be misunderstanding the CLI tool, but I asked a little more detail in a separate discussion.

https://github.com/immich-app/immich/discussions/819#discussion-4468183
